MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -74,6 +74,20 @@ public class DistanceFormatter: LengthFormatter {
                 return 50
             }
             return distance < 3_000 ? 0 : 0.5
+        } else if numberFormatter.locale.identifier == "en-GB" {
+            if unit == .yard {
+                if distance.miles >= 0.1 {
+                    return 0
+                } else if distance.yards < 20 {
+                    return 10
+                } else if distance.yards < 100 {
+                    return 25
+                } else {
+                    return 50
```

Comment:
@frederoni what's your thought about assigning these values (magic numbers) `0, 10, 25, 50` into constant variables? (This would make it more self-documenting is we chose to go with the constant variable(s) approach.)

Reply:
Agreed. I wonder if we can add a rule to the linter about not allowing inline magic values?